### DISCUSSION TITLE
why brute force is not working in https ?

### DISCUSSION THREAD

**Discussion by aymena80s545** [community/community#140708] [Programming Help] [answered]
### Select Topic Area

Question

### Body

i do tools brute force in python and i used tools brute force but they doesn't work i think because https and tls encrypte do you have solution tell me bro?

**Comment by hamzamaimi** [comment]
I don't think there is correlation about brute force attacks and https. Why https should be the reason ? Do you have any log or any error to show ?

**Reply by aymena80s545** [reply]
for example when you do brute force in site you should select the label like username and password but she doesn't have it password , yes she has a password but he is encrypted i speak in request to login you can see this in burp suite , So password data request is encrypted you can't do brute force!!

**Comment by fn-devX** [comment, marked as answer]
to brute force encrypted passwords, you need to reverse-engineer the site's javascript encryption using tools like ```burp suite``` or automate the login with a browser using tools like ```selenium``` or ```playwright```, which handle the encryption for you, try it

**Comment by rabbive** [comment]
Brute force attacks can be challenging with HTTPS/TLS encryption. Make sure you're using tools that support HTTPS requests, like those using Python's requests library. Also, be aware of security measures like rate limiting on target sites. Always ensure you have permission to test systems to stay within legal and ethical boundaries.
